### ISSUE TITLE
Failure to correctly open csv files with carriage returns in text fields.

### ISSUE THREAD

**Issue by 1212Mike** [notepad-plus-plus/notepad-plus-plus#3843]
### Description of the Issue

I have a database that needs to be opened in CSV format for editing.  Some of the text fields contain more than one paragraph thus at least one carriage return.  Although the carriage return is within the text field and should be ignored if the most standard CSV definition is used, notepad plus plus treats it as the beginning of a new record and so disrupts the database.  Apache OpenOffice correctly opens the file for me if I tell it to use double quotes (") but creates the same error if I tell it to use a single quote (').

### Steps to Reproduce the Issue

1. Get a csv file with at least one field containing a carriage return.  (I could supply one but can't find how to upload it - sorry I am new here.)
2. Try opening it in Notepad plus plus (Windows 10)
3. 

### Expected Behavior

The file to open with a new line for each database record.
See attached screenshot "OpenOffice8lines.JPG",
and the screenshot that shows the actual fields with carriage returns - "OpenOffice showing fields with CR.JPG".

### Actual Behavior

An extra line appears in Notepad plus plus for each carriage return that occurs in a text field.
So my dummy database has 8 records but shows up in Notepad plus plus as 10 records (11 lines with the title line).  See screenshot "Notepadplusplus 11 lines.JPG".

### Debug Information

Notepad++ v7.1   (32-bit)
Build time : Oct 15 2016 - 23:30:30
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10
Plugins : mimeTools.dll NppExport.dll NppFTP.dll PluginManager.dll 
![openoffice8lines](https://user-images.githubusercontent.com/33065269/31954204-d75811fe-b8dc-11e7-9586-b290fcfbe36c.JPG)
![openoffice showing fields with cr](https://user-images.githubusercontent.com/33065269/31954202-d72184c2-b8dc-11e7-82f3-346035eaf03e.JPG)
![notepadplusplus 11 lines](https://user-images.githubusercontent.com/33065269/31954203-d73b2a4e-b8dc-11e7-9af8-209f5bb90f54.JPG)

As I am new here please feel free to tell me if there is any more I can do to help you address this bug and make Notepad Plus Plus even better.

Thanks for listening.

**Comment by sasumner** [comment]
Question for the developers:  In a text editor, don't line-ending characters, of whatever form (CR+LF, CR, LF), trump all other definitions?  In other words, if these sequences occur, aren't they considered line-endings no matter what other contextual sequencing happens to be there?

**Comment by 1212Mike** [comment]
An good point @sasumner.  I had been considering the potential damage done to the file if it is then saved.  Some apps I have used have displayed in a similar way when opening and then treated the carriage returns from the text fields in a different way when saving.  But worrying about just this point I have done some further tests and found that Notepad plus plus actually saves the data in exactlly the original format, not causing me any problems,  :-) 

I had been confused by the screen display layout and once I realised that in Notepad plus plus the line end is merely a visual display of the CR character my problems vanished.

So, not so much problem solved, as the problem was only ever in my understanding.  As soon as I can work out how to do so I will be marking this thread resolved,  Sorry for wasting your time.  But thanks for giving me the chance to raise the issue in such a constructive environment.
